### PR TITLE
Add Y20+ series to the batch list

### PR DESCRIPTION
**Pull request by dryairship**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -20,13 +20,15 @@
     "@angular/common": "^5.0.0",
     "@angular/compiler": "^5.0.0",
     "@angular/core": "^5.0.0",
+    "@angular-devkit/core": "^0.0.29",
```
Comment (yashsriv) [inline frontend/package.json:23]: why were these 2 dependencies added?

File: frontend/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
-
-WORKDIR /src
-ADD . /src
-RUN cd /src && yarn build:deploy
-
+### STAGE 1: Build ###
+FROM node:6.16-alpine AS build
+WORKDIR /usr/src/app
+COPY package.json ./
+RUN npm install
```
Comment (yashsriv) [inline frontend/Dockerfile:5]: Why did we get rid of yarn? 😢 I used to like it a lot. Way faster than npm. Plus it had its own yarnfile which won't be respected for the build if you use npm?

**Review by yashsriv — APPROVED** [review]
Overall lgtm :+1:

File: frontend/package.json (hunk 1 above)
Comment (dryairship) [reply frontend/package.json:23]: It raised some errors without these dependencies:
```
module.js:478
    throw err;
    ^

Error: Cannot find module '@angular-devkit/core'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/src/app/node_modules/@angular-devkit/schematics/src/tree/virtual.js:10:16)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
```

Comment (yashsriv) [reply frontend/package.json:23]: Which is why you use yarn which ensures exact dependencies as the one which previously compiled this project are used :P

Comment (dryairship) [reply frontend/package.json]: @yashsriv Okay, I reverted back to using yarn :p

Comment (abdatta) [reply frontend/package.json]: @yashsriv Doesn't the `package-lock.json` file created by npm do the same thing as `yarn.lock` now? Is it required anymore really? @yashsriv

File: frontend/Dockerfile (hunk 2 above)
Comment (abdatta) [reply frontend/Dockerfile]: The exact dependencies are now respected by npm's `package-lock.json` file. So I don't think this should be an issue.

File: frontend/package.json (hunk 1 above)
Comment (yashsriv) [reply frontend/package.json]: @abdatta Technically not. But this is like a 3-4 year old project still using angular 5. If someone brings things up-to-date, they can get rid of yarn/yarn.lock and shift to using npm+package-lock.json.

File: frontend/Dockerfile (hunk 2 above)
Comment (yashsriv) [reply frontend/Dockerfile]: But it won't respect the exact dependencies of the existing yarn.lock file right. It needs to be once built with npm and then the npm package-lock has to be committed.

Comment (abdatta) [reply frontend/Dockerfile]: Hmm, you're right... unless one uses npm 7.0 which respects `yarn.lock` versions as well.

File: frontend/package.json (hunk 1 above)
Comment (abdatta) [reply frontend/package.json]: Makes sense. I created an issue to upgrade the angular and other packages of the frontend. Once that's done, we can hopefully reduce one devDependency :P

**Pull request merged**